vmm/sandbox/src/main.rs

```diff
@@ -100,11 +100,14 @@ pub const NAMESPACE_CGROUP: &str = "cgroup";
 
 pub const FS_SHARE_PATH: &str = "shared_fs";
 
+pub const CONFIG_STRATOVIRT_PATH: &str = "/var/lib/kuasar/config_stratovirt.toml";
```

Comment:
It seems the constant defined here has no use anywhere?

Reply:
Fix it!